Source/Core/Core/WatchDog.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "Core/WatchDog.h"
+
+#include "Common/Assert.h"
+#include "Common/ChunkFile.h"
+#include "Common/Logging/Log.h"
+#include "Common/SPSCQueue.h"
+
+#include "VideoCommon/VideoConfig.h"
+
+#include <algorithm>
+#include <cmath>
+
+WatchDog::Profiler g_watchdog;
```

Comment:
These two should probably still be in the watchdog namespace, rather than being put into the global namespace